common/app/routes/challenges/utils.js

```diff
@@ -46,15 +46,35 @@ export function createTests({ tests = [] }) {
     });
 }
 
+function logReplacer(value) {
+  if (typeof value === 'string' && !value.startsWith('//')) {
+    return '"' + value + '"';
+  } else if (typeof value === 'number' && isNaN(value)) {
```

Comment:
I usually do that. This was originally inside of one of the `map`s setting a variable instead. For some stupid reason I forgot to change it when I refactored it into a function. Will make the change now.